rpc/cloudagent/compute/virtualmachine/moc_cloudagent_virtualmachine.proto

```diff
@@ -155,7 +155,7 @@ service VirtualMachineAgent {
 	rpc RunCommand(VirtualMachineRunCommandRequest) returns (VirtualMachineRunCommandResponse) {}
 }
 
-message HttpProxyConfiguration {
+message ProxyConfiguration {
```

Comment:
Can we declare it in common, since it is used by cloudagent and nodeagent

Reply:
done!